sympy/physics/mechanics/body.py

```diff
@@ -224,11 +241,83 @@ def apply_torque(self, vec):
             >>> body = Body('body')
             >>> T = Symbol('T')
             >>> body.apply_torque(T * body.frame.z)
+
+        Notes
+        =====
+
+        If a new torque is applied on body which already has some torque applied on it,
+        then the new torque is added to the previous torque about the body's frame.
+
         """
 
         if not isinstance(vec, Vector):
             raise TypeError("A Vector must be supplied to add torque.")
-        self.loads.append((self.frame, vec))
+        for load in self._loads:
+            if self.frame in load:
+                vec += load[1]
+                self._loads.remove(load)
+                break
+        self._loads.append((self.frame, vec))
+
+    def clear_load(self):
+        """
+        Clears the Body's loads list.
+
+        Example
+        =======
+
+        >>> from sympy.physics.mechanics import Body
+        >>> B = Body('B')
+        >>> force = B.x + B.y
+        >>> B.apply_force(force)
+        >>> B.loads
+        [(B_masscenter, B_frame.x + B_frame.y)]
+        >>> B.clear_load()
+        >>> B.loads
+        []
+
+        """
+
+        self._loads = []
+
+    def remove_load(self, about):
```

Comment:
This seems potentially useful. If you have many loads acting on the same point you can just say "remove all forces from this point". Maybe this shoudl be `remove_loads_from(point|frame)`.

Reply:
This function would only be useful if `.loads` is immutable.

Reply:
That's not true. This function can only work if `.load` is assume mutable. You can remove something from something that is immutable.

Reply:
If we define `self._loads = [ ]` , and make `loads` a property as I have done in this PR, user can't alter loads of bodies directly but we can.